docs/source/search-syntax.rst

```diff
@@ -72,13 +73,47 @@ For example:
   metrics."2019-04-02 error rate"
 
 
+Run attributes
+~~~~~~~~~~~~~~
+
+Syntax supports searching runs using two attribute names ``status`` and ``artifact_uri``. Search
+will error out if other attribute names used in filter string. Both these attributes have string
+values.
+
+Experiment ID field is already a part of Search API. Run's ``lifecycle_stage`` attribute is not
+allowed in search because it is already encoded as a part of API's ``run_view_type`` field. To
+search for runs using ``run_id`` it is more efficient to use get run APIs. Other fields in
+:py:class:`mlflow.entities.RunInfo` are `System tags`_.
+
+Currently, search does not support ``start_time`` and ``end_time`` attributes, since values have
+datetime datatype.
+
+System tags
```

Comment:
I think @tomasatdatabricks was working on producing a comprehensive doc of the system tags. We may want to link to that instead (as it might be on a different page).

Reply:
Actually went ahead and opened #1342 to add these docs.